core/import_data/import_progress_reports.py

```diff
@@ -0,0 +1,124 @@
+import csv
+import logging
+
+from django.conf import settings
+
+from core.import_data.utils import get_object_by_code
+from core.import_data.utils import get_object_by_name
+from core.import_data.utils import parse_date
+from core.models import Agency
+from core.models import Country
+from core.models import Project
+from core.models import Region
+from core.models.project import ProjectProgressReport
+from core.models.project import ProjectSector
+from core.models.project import ProjectStatus
+from core.models.project import ProjectType
+
+logger = logging.getLogger(__name__)
+
+
+def import_progress_reports():
```

Comment:
I believe there's no need to import data for fields like country, agency, type, and sector that we already have in the project. Instead, we can compare for any differences and display a warning if any are found.

Reply:
Sure. For what is worth, I DID already find examples with differences. Which is why I opted to store duplicate fields that are already linked to the project here. But if a warning is sufficient that's alright with me!